src/test/java/ca/team2706/frc/robot/SendablesTest.java

```diff
@@ -187,6 +187,17 @@ public void newPigeonSendableUpdateTest(@Injectable PigeonIMU pigeon) {
         }};
     }
 
+    private static Delegate makePigeonExpectation(double returns) {
```

Comment:
The variable name `returns` is a little confusing here. Maybe something like `expectedValue`?